components/avatar/avatar.css

```diff
@@ -0,0 +1,25 @@
+.avatar {
+    height: var(--sp-4);
+    width: var(--sp-4);
+    border-radius: 50%;
```

Comment:
Even for `border-radius` , you can have classes like `.rounded` or have a variable. You will using it in badges , images and in many places.

Reply:
Thanks, I will do that.